src/views/tabs/bordered.blade.php

```diff
@@ -0,0 +1,68 @@
+<div class="container py-8 font-sans">
+    <div
+        x-data="{ ...borderedTabs() }"
+    >
+        <ul role="tablist" class="-mb-px z-10 flex items-end">
+            @foreach($tabs as $tab)
+            <li role="presentation" class="inline-flex">
+                <button
+                    id="tab-{{ $loop->index }}"
+                    class="px-6 py-3 font-bold border-t border-l border-r border-transparent hover:border-gray-300 focus:outline-none focus:ring focus:ring-blue-500"
+                    role="tab"
+
+                    x-ref="tab-{{ $loop->index }}"
+                    x-bind:tabindex="tabIndex({{ $loop->index }})"
+                    x-bind:class="tabClasses({{ $loop->index }})"
+                    x-bind:aria-selected="isActiveTab({{ $loop->index }})"
+
+                    x-on:keydown.arrow-left="setActiveTab({{ $loop->index - 1 }})"
+                    x-on:click.prevent="setActiveTab({{ $loop->index }})"
+                    x-on:keydown.arrow-right="setActiveTab({{ $loop->index + 1 }})"
+                >
+                    Section 1
+                </button>
+            </li>
+            @endforeach
+        </ul>
+
+        <div class="border border-gray-300">
+            {{ $slot }}
+        </div>
+
+    </div>
+</div>
+
+<script>
+    const borderedTabs = () => {
```

Comment:
I think we should probably use `var` for these just in case someone puts multiple on one page.

Reply:
Hey @jasonlbeggs , let's merge first #5 to have all scripts in a single place. What do you think? I can move all AlpineJs components there.

Reply:
Pinging for #5 PR to be merged first 👍